### PR TITLE
Merge Lay on Hands into one spell

### PR DESCRIPTION
**Pull request by LebombJames**
May need a migration or 68

### CODE REVIEW

**Comment by LebombJames** [comment]
Due to the deletion of the undead version of the spell and effect, some migration is needed to not break the spell on NPCs. Should be done as part of #1956

File: packs/data/spell-effects.db/spell-effect-lay-on-hands.json [hunk 1]
```diff
@@ -17,10 +18,26 @@
         },
         "rules": [
             {
+                "hideIfDisabled": true,
                 "key": "FlatModifier",
+                "predicate": [
+                    "self:mode:living"
```
Comment (stwlam) [inline packs/data/spell-effects.db/spell-effect-lay-on-hands.json:24]: This should predicate on what the target is rather than the caster

Comment (stwlam) [reply packs/data/spell-effects.db/spell-effect-lay-on-hands.json:24]: Both also need to pivot on negative healing.

Comment (LebombJames) [reply packs/data/spell-effects.db/spell-effect-lay-on-hands.json:24]: How exactly does negative healing interact with it? The healing would be affected, but that's taken care of by the spell variants. How does that relate to the AC bonus/penalty?

I imagine the predicates would be `living` OR NOT `negative healing`, and `undead` OR `negative healing`?

**Pull request closed without merging**